src/cmd/linuxkit/moby/images.go

```diff
@@ -39,16 +37,5 @@ func imagePull(ref *reference.Spec, alwaysPull bool, trust bool, cacheDir string
 // imageLayoutWrite takes an image name and pulls it down, writing it locally
 func imageLayoutWrite(cacheDir string, ref *reference.Spec, architecture string, trust bool) (ImageSource, error) {
```

Comment:
We removed how this uses `trust`, but kept it as an arg. I think we should just get rid of it here, too.